src/internal_modules/roc_audio/latency_tuner.h

```diff
@@ -235,16 +271,35 @@ class LatencyTuner : public core::NonCopyable<> {
     bool has_e2e_latency_;
     packet::stream_timestamp_diff_t e2e_latency_;
 
-    bool has_jitter_;
-    packet::stream_timestamp_diff_t jitter_;
+    bool has_metrics_;
+    LatencyMetrics latency_metrics_;
+    packet::LinkMetrics link_metrics_;
 
+    bool auto_tune_;
     packet::stream_timestamp_diff_t target_latency_;
     packet::stream_timestamp_diff_t min_latency_;
     packet::stream_timestamp_diff_t max_latency_;
     packet::stream_timestamp_diff_t max_stalling_;
 
     const SampleSpec sample_spec_;
 
+    enum TargetLatencyState {
+        TL_NONE,
+        TL_START,
+        TL_INC_TIMEOUT,
+        TL_DEC_TIMEOUT
+    } target_latency_state_;
```

Comment:
Could you add comments for each state?

Also, names are not very descriptive, especially TL_NONE. TL_STARTING would be better than TL_START, and TL_INC_TIMEOUT may be something like TL_COOLDOWN_AFTER_INCREASE or TL_COOLDOWN_AFTER_INC?